### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by AliTaee**

### CODE REVIEW

File: __test__/Pages/about.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import About from '../../pages/about';
+
+describe('Test for about page', () => {
+  it('Should renders about page without crashing', () => {
```
Comment (sohrabtaee) [inline __test__/Pages/about.test.js]: You are not testing whether it crashes or not, you are testing if it shows the title of the page. So, please change the test description to something more meaningful.

File: __test__/src/components/note/note-list-childs/NoteList.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            title: 'unit-test',
+            content: 'Learning Unit Test',
+            date: '2020/03/04',
+            id: 'FTSTSrn7',
+            done: false,
+          },
+        ]}
+      />,
+    );
+    expect(noteList.find('[data-test="no-note-yet"]')).toHaveLength(0);
```
Comment (sohrabtaee) [inline __test__/src/components/note/note-list-childs/NoteList.test.js]: You should check if it exists or not, not its length.

File: src/components/note/ViewNotes.jsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ import { bindActionCreators } from 'redux';
 import { activeMain } from '../../redux/actions/actionMain';
 
 // Components
-import NoteList from './viewChildComponents/NoteList';
+import NoteList from './note-list-childs/NoteList';
```
Comment (sohrabtaee) [inline src/components/note/ViewNotes.jsx]: 💅 Plural for child is children

File: src/components/note/note-list-childs/NoteList.jsx [hunk 4]
```diff
@@ -52,52 +52,56 @@ const NoteList = props => {
...
-                edge="end"
-                aria-label="delete"
-              >
-                <DeleteIcon />
-              </IconButton>
-              <Link
-                href={`/edit?id=${item.id}`}
-                as={`${process.env.BACKEND_URL}/edit?id=${item.id}`}
-              >
+        {tasks.length !== 0 ? (
```
Comment (sohrabtaee) [inline src/components/note/note-list-childs/NoteList.jsx]: you can change it to simply `tasks.length`.

File: __test__/Pages/about.test.js (hunk 1 above)
Comment (AliTaee) [reply __test__/Pages/about.test.js]: Done.

File: src/components/note/ViewNotes.jsx (hunk 3 above)
Comment (AliTaee) [reply src/components/note/ViewNotes.jsx]: Done.

File: src/components/note/note-list-childs/NoteList.jsx (hunk 4 above)
Comment (AliTaee) [reply src/components/note/note-list-childs/NoteList.jsx]: Nice! I'm not knowing we could do that! Done.

File: __test__/src/components/note/note-list-childs/NoteList.test.js (hunk 2 above)
Comment (AliTaee) [reply __test__/src/components/note/note-list-childs/NoteList.test.js]: Done.

**Pull request merged**